### PR TITLE
Added docs link for wfm_project

### PR DESCRIPTION
**Pull request by JameelB**
## Issue
No link to getting started guide for raincatcher project template

## Solution
Add docs field with link to getting started guide for raincatcher project template. 

## JIRA Ticket
https://issues.jboss.org/browse/RHMAP-11343

### CODE REVIEW

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:

* The code changes conform to the contributing & code style guides for this component (if they exist)
* A component owner should review your change (we've tagged these above)
* Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
* If updating server component this PR should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
* Ensure JIRA is up to date before & after merge (e.g. deployment notes, tester notes, release notes, fix version)
* Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.
Fhcap PR will be generated automatically.
To deploy your changes to a cluster place comment in this PR with pattern [x][y] where x=deploy and y=clustername you wish to deploy. 
You can also deploy your change manually using fhcap-cli

**Comment by witmicko** [comment]
needs rebase, LGTM otherwise

**Comment by JameelB** [comment]
retest this please

**Comment by TommyJ1994** [comment]
@JameelB Missing Shrinkwrap version bump. :)

File: global.json [hunk 1]
```diff
@@ -619,6 +619,7 @@
       "priority": 0.30,
       "name": "WFM Demo Project",
       "description": "A demo project showcasing the reusable WFM modules. Note: Using openshift.feedhenry.com access via your OpenShift account is limited to using the small OpenShift Gear. Due to WFM template's non-trivial functionality you may have stability issues using the small gear.",
+      "docs": "https://github.com/feedhenry-staff/wfm-doc/blob/master/Getting-Started.md",
```
Comment (vchepeli) [inline global.json]: should not it point to raincatcher-doc?

Comment (nialldonnellyfh) [inline global.json]: @JameelB @vchepeli yes, it should link to the getting started guide directly as this is the relevant one for users importing the project template.

File: global.json [hunk 2]
```diff
@@ -618,7 +618,7 @@
       "id": "wfm_project",
       "priority": 0.30,
       "name": "WFM Demo Project",
-      "description": "A demo project showcasing the reusable WFM modules. Note: Using openshift.feedhenry.com access via your OpenShift account is limited to using the small OpenShift Gear. Due to WFM template's non-trivial functionality you may have stability issues using the small gear.",
+      "description": "A demo project showcasing the reusable WFM modules. Note: Using openshift.feedhenry.com access via your OpenShift account is limited to using the small OpenShift Gear. Due to WFM template's non-trivial functionality you may have stability issues using the small gear. Get Started: https://github.com/feedhenry-staff/wfm-doc/blob/master/Getting-Started.md",
```
Comment (nialldonnellyfh) [inline global.json]: Can it be added as a <a href=""> link?

**Comment by JameelB** [comment]
Retest this please

**Pull request merged**